backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Uni;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/validation")
+public class ValidationEndpoint {
+
+    @Inject
+    ApplicationResources appResources;
+
+    @GET
+    public Uni<Response> validate(@QueryParam("bundle") String bundleName, @QueryParam("application") String applicationName, @QueryParam("eventtype") String eventType) {
+        return appResources.getEventType(bundleName, applicationName, eventType)
+                .onItem()
+                .transform(this::isValid);
+    }
+
+    private Response isValid(EventType e) {
+        if (e != null) {
+            return Response.ok().entity("bundle found").build();
```

Comment:
Bundle found is probably misleading, as the combo of bundle/app/eT needed to match.
Same below.

Reply:
I don't think an entity is needed in case of success.

Reply:
It's not needed. I just wanted to be able to be able to separate between any possible success and the specific one in our case.

Reply:
While it is not needed, it is good to have it. But the text may be misleading.